README.md

```diff
@@ -9,35 +17,52 @@ Collection of awesome resources for [Neutralino](https://neutralino.js.org/)
 - [Community](#community)
 - [Documentation](#documentation)
 - [Extensions](#extensions)
+- [Templates](#templates)
 - [Tools](#tools)
 - [Videos](#videos)
 
 ## Apps
 
-- [LiteMan](https://github.com/codezri/liteman) - _A simple file viewer app written in Neutralinojs._
-- [LitePy](https://github.com/codezri/litepy) - _World's simplest Python editor built with Neutralinojs_
+- [LiteMan](https://github.com/codezri/liteman) - A simple file viewer app written in Neutralinojs
+- [LitePy](https://github.com/codezri/litepy) - World's simplest Python editor built with Neutralinojs
+- [Drums](https://github.com/m00ke5h/Drums) - A Neutralino app for playing drums
+- [SimpleTime](https://github.com/navidmafi/SimpleTime) - A cross-platform timer and stopwatch app with modern design in mind
+- [MyoroEditor](https://github.com/antonkoetzler/MyoroEditor) - A simple text editor using the lightweight API NeutralinoJS
```

Comment:
Thoughts on making this section (and maybe others too) alphabetical?

Reply:
Great idea, I will update the list